test/fuzz/test_runner.py

```diff
@@ -16,11 +16,14 @@
 
 
 def get_fuzz_env(*, target, source_dir):
+    symbolizer_path = "/usr/bin/llvm-symbolizer"
```

Comment:
Yea, the worst that can happen is the problem we already have. Let's see if we can make this more generic.

Reply:
I guess there are also other issues here, as the ASAN job does find the symbolizer, but not at that path:
```bash
AddressSanitizer: reading suppressions file at /ci_container_base/test/sanitizer_suppressions/ubsan
==28779==Using llvm-symbolizer found at: /usr/bin/llvm-symbolizer-17
==28779==AddressSanitizer Init done
Error parsing command line arguments: ==28779==__tls_get_addr: DTLS_Find 0x7f7962720770 2
```

Reply:
Yea, my question is why is it now failing, if it's successfully finding a symbolizer.

Reply:
Ah, it looks like the `verbosity=2` output is breaking things that are parsing output:
```bash
/usr/bin/python3.11 ../test/util/test_runner.py
2023-11-07 15:13:31,154 - ERROR - Error mismatch:
Expected: Error parsing command line arguments: Invalid command 'foo'
```
and the ASAN job here would otherwise be working as expected.